lua/entities/gmod_tardis/modules/sh_lock.lua

```diff
@@ -50,7 +51,11 @@ if SERVER then
 	
 	ENT:AddHook("Use", "lock", function(self,a,c)
 		if self:GetData("locked") and IsValid(a) and a:IsPlayer() then
-			a:ChatPrint("This TARDIS is locked.")
+			if self:GetData("repairing") then
```

Comment:
This should instead call a more generic hook like DoorOpenLockedAttempt (awful name but you get the idea) and then hook it from within the health module